RELEASING.md

```diff
@@ -0,0 +1,39 @@
+# Release process
+
+## Steps
+
+1. Checkout the repository on the correct branch and changeset (`master`)
+2. Run pre-release script
+
+    1. Release candidate
+
+       In case of release candidate, run localy commmand `make VERSION=x.v.z RELEASE_CANDIDATE=1 pre-release`
+
+       For example `make VERSION=0.3.0 RELEASE_CANDIDATE=2 pre-release` will generate the release version `0.3.0-rc.2`
+
+    2. Final release
+
+       Run localy the command `make VERSION=x.v.z pre-release`
+
+3. Commit all the changes generated from the previous command:
+
+   ```console
+   $ git add .
+   $ git commit -s -m "release vX.Y.X"
```

Comment:
I think you mean `git commit -S` instead of the lowercase `-s`

Reply:
both work on my machine :)

Reply:
They do different things. Lowercase does the "Signed-off-by", while uppercase does GPG signing. If both work for you, it might be because you have `commit.gpgsign` set to true (either on the repo, or on your global config), which might not be the case for everyone.

Reply:
right